src/components/Grids/Grids.tsx

```diff
@@ -93,20 +124,32 @@ export default function Grids({cleanTransactions}: {cleanTransactions: iTransact
   }
 
   function TotalCells({group}: {group: string}) {
-    const catList = Object.keys(gridRowModel[group]);
+    if(gridData[`Total ${group}`] === undefined) {
+      return <></>;
+    }
+    else {
+      return (
+        <>
+        {
+          gridData[`Total ${group}`].map(amount =>  <TableCell>{amount}</TableCell>)
```

Comment:
il ne doit pas être trop content que tu n'as pas de key sur les cells. Note que tu pourrais retourner simplement l'array, react arrive à s'en occuper (sauf erreur de ma part)

Reply:
Keys added to all listed components.